.github/workflows/cleanup.yml

```diff
@@ -9,9 +9,9 @@ jobs:
   cleanup:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
     - name: Configure AWS credentials
-      uses: aws-actions/configure-aws-credentials@v1-node16
+      uses: aws-actions/configure-aws-credentials@v1-node18
```

Comment:
```suggestion
      uses: aws-actions/configure-aws-credentials@v4
```

Reply:
This might fix the failing test. Can you commit the change and verify, @kvegesna?